charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -221,16 +221,29 @@ tests:
       # -- Overrides the image tag
       tag: 5.2.15
 
+  toolkit:
+    image:
+      # -- The OCI registry to pull the tests image from
+      registry: cgr.dev
+      # -- The repository within the registry
+      repository: chainguard/slim-toolkit-debug
+      # -- The tests image pull policy
+      pullPolicy: IfNotPresent
+      # -- This value is deprecated in favor of tag. (Will be removed in a future release)
+      version: ""
+      # -- Overrides the image tag
+      tag: 5.2.15
+
 tools:
   kubectl:
     image:
       # -- The OCI registry to pull the image from
-      registry: docker.io
+      registry: cgr.dev
       # -- The repository within the registry
-      repository: rancher/kubectl
+      repository: chainguard/kubectl
```

Comment:
Earlier we purposefully switched away from the chainguard image to rancher so we could use kubectl versions that matched the host cluster. Chainguard only supports one version of kubectl. Anyway to do this with the rancher version, if the agent route isn't viable?

Reply:
Not unless I install bash or sh

Reply:
reverted to original image with busybox hack